eth/api_tracer.go

```diff
@@ -597,22 +598,17 @@ func (api *PrivateDebugAPI) standardTraceBlockToFile(ctx context.Context, block
 	// Therefore, it's perfectly valid to specify `"futureForkBlock": 0`, to enable `futureFork`
 
 	if config != nil && config.Overrides != nil {
-
-		// TODO(meowsbits): FIXME.
-
-		/*
-
-			// Copy the config, to not screw up the main config
-			// Note: the Clique-part is _not_ deep copied
-			chainConfigCopy := new(params.ChainConfig)
-			*chainConfigCopy = *chainConfig
-			chainConfig = chainConfigCopy
-			if yolov2 := config.Overrides.YoloV2Block; yolov2 != nil {
-				chainConfig.YoloV2Block = yolov2
-				canon = false
+		// Copy the config, to not screw up the main config
+		// Note: the Clique-part is _not_ deep copied
+		chainConfigCopy := new(ctypes.ChainConfigurator)
+		*chainConfigCopy = chainConfig
+		chainConfig = *chainConfigCopy
+		if eip2929 := config.Overrides.GetEIP2929Transition(); eip2929 != nil {
+			if err := chainConfig.SetEIP2929Transition(eip2929); err != nil {
```

Comment:
So we have to use `config.Overrides.GetEIP2929Transition(); eip2929 != nil` to see if YoloV2 is activated (using an indicator EIP as a proxy for all associated EIPs for that fork.

But don't we have to do `chainConfig.SetEIP2929Transition(eip2929)` for _all_ the EIPs (ie `chainConfig.SetEIP2315Transition(eip2929)`, etc.)?

Reply:
Good one. Yes we have to handle all Yolo EIPs this way individually, I am not sure if we should use an indicator EIP to read the fork block number and use it to set all the EIPs on this block. The reason is that on some chain we might want to include only some of the EIPs and not all.

I will add `EIP2315` and `EIP2537` for now.

Reply:
The overrides were actually used for producing an analysis for EIP-2929 and not being used in tests or anywhere else in the code for now. You can check [here](https://github.com/ethereum/go-ethereum/pull/21509#issue-477308710)

Reply:
Resolved with https://github.com/etclabscore/core-geth/pull/246/commits/75af684928f0041d9c5df88776f3e956a508eda4

Reply:
I am re-thinking of it @meowsbits .
1. What they wanted to test was actually only the EIP2929 implications, so the rest can be removed
2. We can even remove this block from here and consider it tested. Later on we can bring it back if we need to test similar E[C]IPs

Reply:
SGTM. Nice investigation!

Reply:
@meowsbits so **leave the code as is** or go with **1** or **2**. I prefer 1.

Reply:
Yes, 1 SGTM.  In general I want to own and persist a superset of the tests from ethereum/go-ethereum (ie. never _only_ rely on them to have tested something, we need to own all the tests... and more ;).

Reply:
Resolves with https://github.com/etclabscore/core-geth/pull/246/commits/26b814975e1c7cdf3afb9f81756c1c0f60446f21.
Which reverts the previous resolving commit :)